src/main/java/com/fasterxml/jackson/databind/DeserializationContext.java

```diff
@@ -169,6 +176,7 @@ protected DeserializationContext(DeserializerFactory df,
             cache = new DeserializerCache();
         }
         _cache = cache;
+        _injectablesCache = new ConcurrentHashMap<>();
```

Comment:
Need not be thread-safe, single deserialization calls are never concurrent.

Also: should not eagerly construct `Map` since this imposes cost for all usage, even when (like in most cases I'd guess) no injection is required.
So should instead lazily construct when needed.

All of this assuming we actually want caching. Not sold on that yet either.